kube-core/src/resource.rs

```diff
@@ -6,6 +6,12 @@ use k8s_openapi::{
 
 use std::{borrow::Cow, collections::BTreeMap};
 
+pub use k8s_openapi::{ClusterResourceScope, NamespaceResourceScope, ResourceScope, SubResourceScope};
+
+/// Indicates that a [`Resource`] is of an indeterminate dynamic scope.
+pub struct DynamicScope {}
```

Comment:
Should this be `DynamicResourceScope` to align with `NamespaceResourceScope` and `ClusterResourceScope`?

Reply:
Good spot. Fixed in https://github.com/kube-rs/kube-rs/pull/956/commits/4af062bde24999b7b6bd5d317ae03a2d9b1d4e76